ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/filter.vrfs.yml

```diff
@@ -0,0 +1,45 @@
+type: l3leaf
+
+l3leaf:
+  nodes:
+    - name: filter.vrfs
+      id: 1
+      loopback_ipv4_pool: 10.0.0.0/24
+      bgp_as: 65001
+      vtep: false
+      filter:
+        deny_vrfs: [VRF1, VRF4]
+        allow_vrfs: [VRF1, VRF2, VRF4, VRF5]
```

Comment:
I understand why VRF1 is in both BUT probably would be good to state that it is to exercise the second part of the AND logic in our filtering function (the test is good just it is not clear why there is allow + deny ;)